graal/backends/core/colic.py

```diff
@@ -59,9 +61,9 @@ class CoLic(Graal):
     :raises RepositoryError: raised when there was an error cloning or
         updating the repository.
     """
-    version = '0.4.0'
+    version = '0.5.0'
 
-    CATEGORIES = [CATEGORY_COLIC_NOMOS, CATEGORY_COLIC_SCANCODE]
+    CATEGORIES = [CATEGORY_COLIC_NOMOS, CATEGORY_COLIC_SCANCODE, CATEGORY_COLIC_SCANCODE_CLI]
```

Comment:
This line seems to long, maybe it could be rewritten as:
```
CATEGORIES = [CATEGORY_COLIC_NOMOS, 
              CATEGORY_COLIC_SCANCODE, 
              CATEGORY_COLIC_SCANCODE_CLI]
```
What do you think ?